resources/qml/WelcomePages/WhatsNewContent.qml

```diff
@@ -21,39 +25,170 @@ Item
+                        top: parent.top
+                        margins: UM.Theme.getSize("default_margin").width / 2
+                    }
+                    color: UM.Theme.getColor("viewport_overlay")
+
+                    Image
+                    {
+                        id: subpageImage
+
+                        anchors
+                        {
+                            horizontalCenter: parent.horizontalCenter
+                            top: parent.top
+                            topMargin: UM.Theme.getSize("default_margin").width
+                        }
+                        width: parent.width - (UM.Theme.getSize("default_margin").width * 2)
+                        height: (parent.height - (UM.Theme.getSize("default_margin").width * 2)) * 0.8
+                        fillMode: Image.PreserveAspectFit
+
+                        source: manager.getSubpageImageSource(index)
+                    }
+
+                    Cura.ScrollableTextArea
+                    {
+                        id: subpageText
+
+                        anchors
+                        {
+                            top: subpageImage.bottom
+                            bottom: parent.bottom
+                            horizontalCenter: parent.horizontalCenter
+                        }
+                        width: parent.width - (UM.Theme.getSize("default_margin").width * 2)
+
+                        ScrollBar.horizontal.policy: ScrollBar.AlwaysOff
 
-        ScrollBar.horizontal.policy: ScrollBar.AlwaysOff
+                        back_color: UM.Theme.getColor("viewport_overlay")
+                        do_borders: false
 
-        textArea.text: CuraApplication.getTextManager().getChangeLogText()
-        textArea.textFormat: Text.RichText
-        textArea.wrapMode: Text.WordWrap
-        textArea.readOnly: true
-        textArea.font: UM.Theme.getFont("medium")
-        textArea.onLinkActivated: Qt.openUrlExternally(link)
+                        textArea.wrapMode: TextEdit.Wrap
+                        textArea.text: manager.getSubpageText(index)
+                        textArea.textFormat: Text.RichText
+                        textArea.readOnly: true
+                        textArea.font: UM.Theme.getFont("medium")
+                        textArea.onLinkActivated: Qt.openUrlExternally(link)
+                    }
+                }
+            }
+        }
     }
 
-    Cura.PrimaryButton
+    PageIndicator
     {
-        id: getStartedButton
-        anchors.right: parent.right
+        id: whatsNewDots
+
+        currentIndex: whatsNewViewport.currentIndex
+        count: whatsNewViewport.count
+        interactive: true
+
+        anchors
+        {
+            bottom: bottomSpacer.top
+            horizontalCenter: parent.horizontalCenter
+        }
+
+        delegate:
+            Rectangle
+            {
+                implicitWidth: UM.Theme.getSize("thin_margin").width
```

Comment:
```suggestion
                width: UM.Theme.getSize("thin_margin").width
```
Implicit width is for when you are creating re-usable components (and defining what it's width should be if nothing is set)